.github/workflows/README.md

```diff
@@ -58,3 +58,10 @@ In order to not fill-up space on the repo with the docs-site related files, the
 - Permissions
 
     The `docs-site` GitHub Action uses a deploy key specifically created for it. To revoke access to the GitHub Action, delete the secret `ACTIONS_DEPLOY_KEY` in [Repository Settings](https://github.com/mlperf/mlbox/settings/secrets/).
+
```

Comment:
Expanding on the scope of this PR, the reason for this document is to empower end-users and new contributors and to ensure that every part of the codebase is approachable. Any developer without explicit knowledge of GitHub Actions should be able to make changes to it in the event that they have to.

- [ ] Could we possibly add documentation around how the publish actions work?
  - Might be helpful in debugging why the Action is failing
- [ ] Create a separate issue on why the Action is failing

Reply:
Not sure Why github action failed. But in future we can capture the error since we enable verbose.